docs/bookmarks.md

```diff
@@ -0,0 +1,311 @@
+# Docs: Bookmarks
+
+Features:
+
+* Associate the bookmarks control with a `MapView` or `SceneView` (`GeoView` property), through binding on supported platforms (WPF, UWP, Forms) or plain properties otherwise.
+* Display a list of bookmarks, defined by `BookmarkList` property or the `Map` or `Scene` from the associated `GeoView`.
+* Navigates the associated `GeoView` to the selected bookmark.
+* Specify whether to display the list from `BookmarkList` or the `Map`/`Scene` `Bookmarks` property using the `PrefersBookmarksList` property.
```

Comment:
I'd remove that property. If you set it, it means you'll be using it. Fall back to map/scene if not set. Name should probably indicates it's an override.
Also remove "List" from name. That's a type but we don't put that in the property name (and type should be ienumerable anyway).